File: 20221102/모범답안/20221102_03/IntersectionTwoArray.py
def solution(arr1, arr2):
    """
    :param arr1: int[]
    :param arr2: int[]
    :return: int[]
    """

    intersection = []

    for i in range(len(arr1)):
        for j in range(len(arr2)):
            if arr1[i] == arr2[j]:
                intersection.append(arr1[i])

    intersection.sort()
    return intersection

File: 20221102/모범답안/20221102_03/IntersectionTwoArray.js
/**
 * @param arr1 {number[]}
 * @param arr2 {number[]}
 * @return {number[]}
 */
function solution(arr1, arr2) {
  const intersection = []

  for (let i = 0; i < arr1.length; i++) {
    for (let j = 0; j < arr2.length; j++) {
      if (arr1[i] === arr2[j]) {
        intersection.push(arr1[i])
      }
    }
  }

  intersection.sort((a, b) => a - b)
  return intersection
}

export default solution
